### PR TITLE
Time conductor real time 4914

### PR DESCRIPTION
**Pull request by afahey87**
Closes 
#4914 

### Describe your changes:

Created test: Time conductor input fields real-time mode
The test Clicks on realtime "Local Clock" mode and updates the start offset and the preceding offset time in seconds. The test validates the new expected offset times. 

### All Submissions:

* [ x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [ x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [ ] Command line build passes?
* [ x] Has this been smoke tested?
* [ x] Testing instructions included in associated issue?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js [hunk 1]
```diff
@@ -67,3 +67,48 @@ test.describe('Time counductor operations', () => {
         expect(endDateValidityStatus).not.toBeTruthy();
     });
 });
+
+
+// Testing instructions:
+// Try to change the realtime offsets when in realtime (local clock) mode.
+test.describe('Time conductor input fields real-time mode', () => {
```
Comment (unlikelyzero) [inline e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js:74]: First of all, this is a perfect implementation. Just some minor nitpicks

File: e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js [hunk 2]
```diff
@@ -67,3 +67,48 @@ test.describe('Time counductor operations', () => {
...
+        await page.locator('.c-conductor__delta-button >> text=00:30:00').click();
+
+        // Input start time offset
+        await page.fill('.pr-time-controls__secs', '23');
+
+        // Click the check button
+        await page.locator('.icon-check').click();
+
+        // Verify time was updated on time offset button
+        const startTimeOffset = page.locator('.c-conductor__delta-button').first();
```
Comment (unlikelyzero) [inline e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js:98]: if this locator won't be used again, we could just do

`await expect(page.locator('.c-conductor__delta-button').first()).toContainText('00:30:23');`

File: e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js [hunk 3]
```diff
@@ -67,3 +67,48 @@ test.describe('Time counductor operations', () => {
...
+        await expect(startTimeOffset).toContainText('00:30:23');
+
+        // Click time offset set preceding now button
+        await page.locator('.c-conductor__delta-button >> text=00:00:30').click();
+
+        // Input preceding time offset
+        await page.fill('.pr-time-controls__secs', '31')
```
Comment (unlikelyzero) [inline e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js:105]: nice! I've not used `.fill` yet

File: e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js (hunk 2 above)
Comment (afahey87) [reply e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js:98]: Thank you, I will make the update.

Comment (afahey87) [reply e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js:98]: Done

**Comment by afahey87** [comment]
@unlikelyzero I have made the suggested updates. Please let me know if I need to make more. Thank you

**Comment by unlikelyzero** [comment]
@afahey87 our pr:e2e trigger is designed to run the entire suite on a PR. Unfortunately, it doesn't work against forks :(

Would you mind running your test in repeat-each mode just to verify that there are no timing issues?
`npm run test:e2e:ci -- timeconductor --repeach-each 10`

If that passes, we're good to go!!!

**Comment by afahey87** [comment]
> @afahey87 our pr:e2e trigger is designed to run the entire suite on a PR. Unfortunately, it doesn't work against forks :(
> 
> Would you mind running your test in repeat-each mode just to verify that there are no timing issues? `npm run test:e2e:ci -- timeconductor --repeach-each 10`
> 
> If that passes, we're good to go!!!

On it!

**Comment by afahey87** [comment]
I am getting an error when I try and run your command:

```
adamfahey@Adams-MacBook-Pro openmct % npm run test:e2e:ci -- timeconductor --repeach-each 10

> openmct@2.0.3 test:e2e:ci
> npx playwright test --config=e2e/playwright-ci.config.js --project=chrome smoke default condition timeConductor "timeconductor" "--repeach-each" "10"

error: unknown option '--repeach-each'
```

Any idea?

**Comment by afahey87** [comment]
Im running the following command ten times npm run test:e2e:ci -- timeconductor

**Comment by afahey87** [comment]
@unlikelyzero Everything looks good!

Here are the results. 

`  Slow test file: [chrome] › visual/default.visual.spec.js (1m)
  Slow test file: [chrome] › plugins/condition/condition.e2e.spec.js (17s)
  Consider splitting slow test files to speed up parallel execution

  12 passed (1m)
::warning title=Slow Test,file=[chrome] › visual/default.visual.spec.js::[chrome] › visual/default.visual.spec.js took 1m
::warning title=Slow Test,file=[chrome] › plugins/condition/condition.e2e.spec.js::[chrome] › plugins/condition/condition.e2e.spec.js took 17s
::notice title=🎭 Playwright Run Summary::  12 passed (1m)
adamfahey@Adams-MacBook-Pro openmct % npm run test:e2e:ci -- timeconductor
`

**Review by unlikelyzero — APPROVED** [review]

**Comment by unlikelyzero** [comment]
@afahey87 great job!

**Comment by afahey87** [comment]
Thanks it was fun!

**Pull request merged**